### PR TITLE
Testnet user doc

### LINKED ISSUE #364: Hadapsar testnet user document

### PR DESCRIPTION
**Pull request by jayeshbairagi**
Fixes #364 
This PR contains user documentation for the Hadapsar testnet

### CODE REVIEW

File: Hadapsar-testnet.md [hunk 1]
```diff
@@ -0,0 +1,617 @@
...
+  Update these constants in the following code:
+  ```js
+  // The amount to be approved
+  const APPROVE_AMOUNT = '<APPROVE_AMOUNT>';
+  // The absolute file path of the keystore file
+  const ACCOUNT_KEYSTORE_FILE_PATH = '<ACCOUNT_KEYSTORE_FILE_PATH>';
+  // The absolute file path of the keystore password
+  const ACCOUNT_PASSWORD_FILE_PATH = '<ACCOUNT_PASSWORD_FILE_PATH>';
+  // The ERC20 contract address for the utility token on metachain(Hadapsar-1405)
+  const ERC20_TOKEN_CONTRACT_ADDRESS = '<TOKEN_ADDRESS>';
```
Comment (deepesh-kn) [inline Hadapsar-testnet.md]: This is ERC20_UTILITY_TOKEN_CONTRACT_ADDRESS.
There should be a code snippet to get the utility token address from the ERC20 value token address.

File: Hadapsar-testnet.md [hunk 2]
```diff
@@ -0,0 +1,617 @@
...
+        data: approveData,
+      });
+
+      const rawTxApprove = {
+        from: accountAddress,
+        nonce: `0x${nonce.toString(16)}`,
+        data: approveData,
+        to: ERC20_TOKEN_CONTRACT_ADDRESS,
+        gasLimit,
+        gasPrice: 10000000000,
```
Comment (deepesh-kn) [inline Hadapsar-testnet.md]: Is this value correct?
Is there any way to not hardcode this value?
(for all `gasPrice` in this document.)

**Review by deepesh-kn — CHANGES_REQUESTED** [review]
Created a PR  https://github.com/jayeshbairagi/facilitator/pull/3 to your branch.
Also, I added few comments in this PR.

File: Hadapsar-testnet.md (hunk 2 above)
Comment (jayeshbairagi) [reply Hadapsar-testnet.md]: Yes, just checked, we can use `web3.eth.getGasPrice()`. Updated the document for all the instances.

File: Hadapsar-testnet.md (hunk 1 above)
Comment (jayeshbairagi) [reply Hadapsar-testnet.md]: If we have a code snippet to get the utility token address from the ERC20 value token address then, should we take `ERC20_VALUE_TOKEN_CONTRACT_ADDRESS` as input and get the utility token address from ERC20Gateway? We will also have to introduce the `ERC20Gateway` address also input parameter for this.
I think it is fine if we take the `ERC20_UTILITY_TOKEN_CONTRACT_ADDRESS`(will update the variable name as suggested above) as input.
Let me know which one to go with

Comment (jayeshbairagi) [reply Hadapsar-testnet.md]: Updated as per discussion

**Review by deepesh-kn — APPROVED** [review]

**Pull request merged**